docs/portal_config.md

```diff
@@ -14,6 +14,9 @@ Below is an example, with inline comments describing what each JSON block config
   "ddEnv": "DEV", // optional; the Datadog RUM option specifying the application’s environment, for example: prod, pre-prod, staging, etc. Can be determined automatically if omitted
   "ddUrl": "", // optional: the Datadog RUM site/url. Defaults to datadoghq.com
   "ddSampleRate": 100, // optional; numeric; the Datadog RUM option specifying the percentage of sessions to track: 100 for all, 0 for none. Default to 100 if omitted
+  "faroEnable": true, // optional; flag to turn on Grafana Faro RUN, default to false
+  "faroUrl": "", // optional: the Grafana Faro collector url. Defaults to https://faro.planx-pla.net/collect
+  "faroSampleRate": 1, // optional; numeric; the Grafana Faro option specifying the percentage of sessions to track: 1 for all, 0 for none. Default to 1 if omitted
```

Comment:
could we organize this more, maybe like this?
(also suggesting "grafanaFaro" instead of just "faro")
```
"featureFlags": {
  "grafanaFaro": true
},
"grafanaFaroConfig": {
  "grafanaFaroUrl": "",
  "grafanaFaroSampleRate": 1
}
```

Reply:
Honestly the way those configs are being setup now is just to make them similar to whatever we have now for other tracking methods (GA/DAP/DD)
In the future when we get rid of DD we will combine/rename some of configs
I think we can put all faro config fields in a section called `grafanaFaroConfig`, but maybe not enabling it from within `featureFlags`

Reply:
oh i see you changed it